.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -0,0 +1,33 @@
+name: Bug Report
+description: Create a report to help us improve SciPy
+title: "[Bug]: "
+labels: [defect]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to file a bug report.
+  - type: markdown
+    attributes:
+        value: Please fill in the fields below, deleting the sections that don't apply to your issue.
+  - type: markdown
+    attributes:
+        value: You can view the final output by clicking the preview button above.
+  - type: input
+    attributes:
+      label: Reproducing Code Example
+      description: 'If you place your code between the triple backticks below, it will be rendered as a code block.'
```

Comment:
The blog-post sounds like `render: shell` should handle the triple backtick problem.  Otherwise, I think you have to actually show it (with newlines), because it should not be necessary to even know what a backtick is.

Reply:
Done!